egs/jsut/asr1/run.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=pytorch
+stage=-1       # start from -1 if you need to start from data download
+stop_stage=100
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+nj=10
+
+# feature configuration
+do_delta=false
+
+lm_config=conf/lm.yaml
+train_config=conf/train.yaml
+decode_config=conf/decode.yaml
+
+lm_resume=
+lmtag=
+# decoding parameter
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+
+# root directory of db
+db_root=./downloads
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+. ./path.sh
```

Comment:
I don't remember why we have `. ./path.sh` and `. ./cmd.sh` twice. I'll put it in the GitHub issue.

Reply:
I'll delete the second path.sh and cmd.sh.

Reply:
You don't have to do it now.
It seems that all recipes have such duplications and maybe there are some reasons.

Reply:
sure.